### DISCUSSION TITLE
Project 1

### DISCUSSION THREAD

**Discussion by AmanuelDagnew** [cooperrc/computational-mechanics#154] [Q&A]
I was finishing up project 1 and I know matplotlib is imported. I was wondering if I needed to add plots. 

Thank you

**Comment by DJA17** [comment]
You can use graphs in 3a. to show that the Euler and Analytical solutions converge. Try decreasing dt (time step) to see this relation. Good luck!

**Comment by AmanuelDagnew** [comment]
Thank you. Happy new year!
